src/main/java/appeng/integration/modules/RF.java

```diff
@@ -69,7 +69,7 @@ public void postInit() {
         this.registerRFAttunement("ThermalDynamics", "ThermalDynamics_0", 6);
 
         this.registerRFAttunement("EnderIO", "itemPowerConduit", OreDictionary.WILDCARD_VALUE);
-        this.registerRFAttunement("EnderIO", "blockCapacitorBank", 0);
+        this.registerRFAttunement("EnderIO", "blockCapBank", 0);
```

Comment:
Currently this only works with the creative one. This allows it to work with all variations.
```suggestion
        this.registerRFAttunement("EnderIO", "blockCapBank", OreDictionary.WILDCARD_VALUE);
```